docs/4.1/admin-guide.md

```diff
@@ -2125,25 +2119,24 @@ To configure Teleport to use DynamoDB:
 teleport:
   storage:
     type: dynamodb
-    region: eu-west-1
+    # Region location of dynamodb instance, https://docs.aws.amazon.com/en_pv/general/latest/gr/rande.html#ddb_region 
+    region: us-east-1
 
     # Name of the DynamoDB table. If it does not exist, Teleport will create it.
-    table_name: teleport_table
-
-    # Authentication settings are optional (see below)
-    access_key: BKZA3H2LOKJ1QJ3YF21A
-    secret_key: Oc20333k293SKwzraT3ah3Rv1G3/97POQb3eGziSZ
+    # table_name is 
+    table_name: Example_TELEPORT_DYNAMO_TABLE_NAME
+    audit_table_name: Example_TELEPORT_DYNAMO_EVENTS_TABLE_NAME
```

Comment:
It turns out `audit_table_name` is deprecated (https://github.com/gravitational/teleport/issues/3071#issuecomment-540162977) so that's why it isn't in the docs. Sorry :(